review-ui/Ahk.Review.Ui/Models/Assignment.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Ahk.Review.Ui.Models
+{
+    public class Assignment
```

Comment:
Ha lesznek DTO-k akkor azokat szervezzük ki egy külön projektbe és hivatkozzuk be a UI-ről és az API-ból is, így kódmegosztás tudunk elérni